recipes/theano-pymc/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "1.0.5" %}
+
+package:
+  name: theano-pymc
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/T/Theano-PyMC/Theano-{{ version }}.tar.gz
```

Comment:
It looks like the `url` is wrong. Should be

```
.../Theano-PyMC-{{ version }}.tar.gz
```

I think

Reply:
Fixed.